ToDo-Garden/ToDo-Garden/HomeScene/Sources/HomeScene/HomeSceneInteractor.swift

```diff
@@ -44,7 +45,13 @@ final class HomeSceneInteractor: HomeSceneDataStore {
   private var homeSceneWorker: HomeSceneWorkable
   private var monthlyData: [String: [SharedEntity.TodoListGroup]]
   // ⬆️ 서버에서 받아오는 1달짜리 데이터입니다. ex) key = "20250302"
-  private var itemsForBatch: [String: TodoBatchItem]
+  private var itemsForBatch: [String: TodoBatchItem] {
+    didSet {
+      itemsForBatch.values.forEach {
+        print($0.localId)
+      }
+    }
+  }
```

Comment:
```suggestion
  private var itemsForBatch: [String: TodoBatchItem] 
```

didSet 블럭만 제거하면 될 것 같습니다